### ISSUE TITLE
Cannot create new Serverpod app with busy port (`launchd` process on port :8090)

### ISSUE THREAD

**Issue by TylerSustare** [serverpod/serverpod#517]
I tried creating a new project using the following commands, when the CLI wouldn't create the project.

I don't know what the CLI would do, so I haven't passed the `-f` option. That would be an easy fix, but this issue is based on the `launchd` process not being in my control and keeping me from creating a new Serverpod app.

I also can't simply `$ kill -9 1` due to it being the `launchd` process. Killing that process kills the computer and restarts it. From my experience 😆 

```
src/github.com/tylersustare
❯ serverpod create superduper
There are some issues with your setup that will prevent your Serverpod project from running out of the box and
without further configuration. You can still create this project by passing -f to "serverpod create" and
manually configure your Serverpod.

One or more network ports Serverpod want to use are not available. The most likely reason is that you have
another Serverpod project running, but it can also be another service.

Ports in use:
 • 8090: PostgreSQL server

You do not have Docker installed or it is not running. Serverpod uses Docker to run Postgres and Redis. It's
recommended that you install Docker Desktop from https://www.docker.com/get-started but you can also install
and configure Postgres and Redis manually and run this command with the -f flag added.

src/github.com/tylersustare
➜ sudo lsof -i :8090
Password:
COMMAND PID USER   FD   TYPE             DEVICE SIZE/OFF NODE NAME
launchd   1 root   11u  IPv4 0xb8c5c6c13b9d1987      0t0  TCP localhost:8090 (LISTEN)
```

**Comment by yahu1031** [comment]
@TylerSustare If it is used by `launchd` natively, then you won't be seeing it as `localhost:8090`. It may be like `*:8090`. Please cross-check docker is up and that you stopped all the containers.

**Comment by TylerSustare** [comment]
Hey @yahu1031 thank you. I have confirmed no containers are running.

**Comment by yahu1031** [comment]
@TylerSustare check out in activity monitor and see if there is any other launchd is running and if it does try killing this launchd which is using 8090.

**Comment by TylerSustare** [comment]
I don't believe there are any other `launchd` processes running. I can try killing this process again though.

> I also can't simply $ kill -9 1 due to it being the launchd process. Killing that process kills the computer and restarts it. From my experience 😆

<img width="726" alt="Screenshot 2022-12-27 at 11 50 07 PM" src="https://user-images.githubusercontent.com/10850753/209777723-7ca53b5c-1efe-4b3a-9f4f-8812f0e9265a.png">

**Comment by TylerSustare** [comment]
Yeah I'm sending this from my phone because killing this process restarted my computer again.

It seems reasonable to pass a port flag to this command.

**Comment by TylerSustare** [comment]
And to update. I tried passing the `-f` flag to create the project anyway and I'm (unsurprisingly) unable to run the project.

```
tylersustare/superduper/superduper_server via 🎯 v2.18.3
➜ docker-compose up --build --detach
[+] Running 1/2
 ⠿ Container superduper_server-postgres-1  Starting                                                       0.4s
 ⠿ Container superduper_server-redis-1     Started                                                        0.4s
Error response from daemon: Ports are not available: exposing port TCP 0.0.0.0:8090 -> 0.0.0.0:0: listen tcp 0.0.0.0:8090: bind: address already in use
```

But still, the only thing running on `:8090` is my `launchd` process. Even after restarting 3 times, that's still the port used  🤷 and I have no idea how to change it.

**Comment by vlidholt** [comment]
If you create your project with the -f flag, you will need to change the configuration files (`config/development.yaml` + the docker container that starts the database) to use other ports, you will also need to manually install the database tables.
